Audacia.Quest/Audacia.Quest.Core/Component/Transform.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Audacia.Quest.Core.Components
+{
+    public class Transform
+    {
+        public Vector2 Position { get; set; } = Vector2.Zero;
+        public Vector2 Scale { get; set; } = Vector2.One;
+        public Vector2 Rotation { get; set; } = Vector2.Zero;
+
+        public void Move(Vector2 other)
+        {
+            Position = other;
+        }
+
+        public void MoveX(float x)
+        {
+            var pos = new Vector2(Position.X + x, Position.Y);
```

Comment:
The temporary variable for `pos` is a bit redundant since no other changes need to occur before setting position.
You may just want to set `Position` directly but, it doesn't matter too much as the extra assignment will be compiled out.